tensorflow/tools/proto_text/BUILD

```diff
@@ -35,6 +35,7 @@ cc_binary(
     deps = [
         ":gen_proto_text_functions_lib",
         "//tensorflow/core:lib_proto_parsing",
+        "//tensorflow/core:lib_internal",
```

Comment:
Oops, you'll need to swap this line with the previous one.  Otherwise our BUILD-file lint checker fails.